Feature/siq file setting

GalievBulat opened the pull request.

niyazm524 committed:
Merge branch 'master' into feature/siq_file_setting

# Conflicts:
#	app/src/main/AndroidManifest.xml
#	app/src/main/java/ru/itis/androidsummer/MainMenuActivity.kt
#	app/src/main/java/ru/itis/androidsummer/SplashActivity.kt
#	app/src/main/res/values/strings.xml

niyazm524 committed:
revert layout of main menu activity due to code conventions issue

GalievBulat committed:
вывод ошибок 145

GalievBulat committed:
все добавил

niyazm524 commented:
// REF: Issue #4 
@GalievBulat Посмотри как принято называть id элементов в layout

niyazm524 reviewed (changes requested):
@GalievBulat 
@ddddi21 
Что-то вы не совместили свои идеи насчет UX приложения в какую-то общую идею, вам нужно вместе сесть и решить как всё должно работать (где настройки, что там есть, когда выбор пака)
P.S. свои идеи я описал в файле `MainMenuActivity.kt`, [загляните](https://github.com/niyazm524/AndroidSummer2020/blob/2c40c85977d0e08a5aef5d500eaf334c017c3aaa/app/src/main/java/ru/itis/androidsummer/MainMenuActivity.kt)

niyazm524 commented:
## + тащить в APK **50 МБ** не очень много желания если честно, подгрузку с файловой системы бы (на крайняк можно и так оставить, но не оч красиво)

ddddi21 commented on app/src/main/java/ru/itis/androidsummer/MainMenuActivity.kt:
```diff
@@ -12,14 +12,26 @@ class MainMenuActivity : AppCompatActivity() {
         iv_main_menu_settings.setOnClickListener {
-            startActivity(Intent(this,SettingsActivity::class.java))        }
+            /*
+            Bulat version: startActivity(Intent(this, PackChoosing::class.java))
```
![W8GXw05I7WI](https://user-images.githubusercontent.com/55531213/87548552-55422580-c6b5-11ea-95e2-417a22131540.jpg)

GalievBulat committed:
внешние паки поддержка

GalievBulat committed:
-cafe

GalievBulat committed:
prefs fix

GalievBulat committed:
naming fix

GalievBulat committed:
naming fix2

GalievBulat committed:
threw out the randomization method

GalievBulat committed:
threw out the randomization method

ddddi21 commented:
Нияз, тебя тоже ставлю в известность об этой находке
![ARGYrpShqSU](https://user-images.githubusercontent.com/55531213/87608317-420c7580-c708-11ea-9aed-008ed74b0d6a.jpg)
в общем там посмотрите, я думаю не нужно для выбора паков отдельное активити, просто перенести весь код в рекуклер в созданный активити из моей ветки
кароче надо было опять обсудить все Булатек ну ты шо, я же предупредила что логику реализую/реализовала

ddddi21 commented:
> Нияз, тебя тоже ставлю в известность об этой находке
> ![ARGYrpShqSU](https://user-images.githubusercontent.com/55531213/87608317-420c7580-c708-11ea-9aed-008ed74b0d6a.jpg)
кароче там посмотрите, я думаю не нужно для выбора паков отдельное активити, просто перенести весь код в рекуклер в созданный активити из моей ветки

niyazm524 committed:
Merge branch 'master' into feature/siq_file_setting

# Conflicts:
#	app/src/main/AndroidManifest.xml
#	app/src/main/java/ru/itis/androidsummer/GameInterfaceActivity.kt
#	app/src/main/java/ru/itis/androidsummer/MainMenuActivity.kt

niyazm524 committed:
onBackPressed stack bug fixed

niyazm524 reviewed (changes requested):
ну всё, осталось только смержить ваши функциональности в одну конфетку

niyazm524 commented:
Я смержил с мастера изменения, так что теперь тут всё самое последнее

GalievBulat committed:
merged with Dilyara

GalievBulat committed:
merged with Dilyara2

GalievBulat committed:
merged with Dilyara4

GalievBulat committed:
Merge remote-tracking branch 'origin/feature/siq_file_setting' into feature/siq_file_setting

# Conflicts:
#	app/src/main/AndroidManifest.xml
#	app/src/main/java/ru/itis/androidsummer/SingleMultiActivity.kt
#	app/src/main/res/layout/activity_single_multi.xml

GalievBulat committed:
merged with Dilyara4

ddddi21 committed:
add levels for ui single player, upgrade SimpleMultiActivity

ddddi21 committed:
from Dilyara
Merge remote-tracking branch 'origin/Add/hard_medium_light' into feature/siq_file_setting

add levels for bot, complete our ui, fix design
probably will be update

ddddi21 committed:
change button's name

niyazm524 committed:
refactoring and using intents instead of shared prefs

niyazm524 reviewed (approved):
Немного переделал способ передачи информации о выбранном паке: [коммит](https://github.com/niyazm524/AndroidSummer2020/pull/56/commits/f47f2e320002a81982cca7f36056ffaa7fc1beba) 
В остальном всё супер
